README.md

```diff
@@ -89,10 +52,10 @@ schema(
 
 ## Installation
 
-To use this package, it is enough to build it.
-It will be exported automatically in the cmake user configuration, so that it can be found using CMake's `find(smc_verifiable_plugins REQUIRED)` macro.
-After running that macro, the variable `smc_verifiable_plugins_INCLUDE_DIR` with the path to the include folder,
-and the variable `smc_verifiable_plugins_PLUGINS_PATH` with the path to the library folder, will be provided, and can be used to locate the existing plugins' libraries.
+If using this package for developing new plugins, it is enough to integrate it in your CMake project following the instructions in [the documentation](https://convince-project.github.io/smc_verifiable_plugins/development.html#fetch-this-package-in-cmake).
+
+For active development of this package, having it build will automatically this package available to the other packages depending on it, that can found this package using CMake's `find(smc_verifiable_plugins REQUIRED)` macro.
+After running that macro, the variable `smc_verifiable_plugins_INCLUDE_DIR` with the path to the include folder, and the variable `smc_verifiable_plugins_PLUGINS_PATH` with the path to the library folder, will be provided, and can be used to locate the existing plugins' libraries.
```

Comment:
```suggestion
After running that, the variable `smc_verifiable_plugins_INCLUDE_DIR` with the path to the include folder, and the variable `smc_verifiable_plugins_PLUGINS_PATH` with the path to the library folder, will be provided, and can be used to locate the existing plugins' libraries.
```

Also: This sentence does not only apply to the case where someone actively develops this package. I assume the variables are also set in the case where people use the `FetchContent_Declare(`-way, right?